decidim-module-census_connector/app/controllers/decidim/census_connector/verifications/census/authorizations_controller.rb

```diff
@@ -20,7 +20,6 @@ def index
               @handler = current_handler.from_model(person).with_context(form_context)
             else
               @handler = current_handler.new.with_context(form_context)
-              @handler.use_default_values
```

Comment:
I'll double check and try to add a test if there's a regression. :+1:

Reply:
I haven't looked yet but since I run into the same issue and thought before, I'll expose it here:

In my opinion, we should split out all data modification logic into separate controller, forms, and views. Current logic is complicated enough to fit all the cases and the data modification has its specifities:

* Different copy.
* Possibly different forms.
* Single form workflow.

Instead of adding more conditionals and code branches to check which situation we're handling and thus whether a field should be displayed or not, or sent to the server or not, I would move all the logic to a separate place. Even if it implies a bit of code duplication.

Reply:
I agree with you. Let's finish merging this PR and think a little more about how to do that.